packages/react/src/components/Icon/Icon.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import IbmLogo from './svg/ibm-logo.svg';
+
+/**
+ * Icon component
+ *
+ * @param {string} type The icon
+ * @param {string} size Set icon size
```

Comment:
I put this in there in case I needed it for testing layout but in the future I don't think we'll need to have something like this. All icons that we provide should be a preset range of sizes.